### PR TITLE
Case-insensitive search (fixes #95, #87)

### LINKED ISSUE #95: Case-insensitive search
The search should not be case-sensitive. For example, when I search for 'public' I get 0 hits but when I search for 'Public' I get the list of Public Cloud Modules.

### PR DESCRIPTION
**Pull request by crazyscientist**
* Add filter options for case-insensitive searching
* Also support filtering by entity name and slug
* Offer advanced filter options in the UI for name and slug

### CODE REVIEW

**Comment by der-gabe** [comment]
> * Also support filtering by entity name and slug
> * Offer advanced filter options in the UI for name and slug

Do we have an issue for that?

**Comment by crazyscientist** [comment]
> Do we have an issue for that?

Nice catch. Yes, we had one: #87

File: backend/crud.py [hunk 1]
```diff
@@ -437,18 +438,24 @@ def _query_entity_with_filters(filters: dict, schema: Schema, all: bool = False,
     '''
     attrs = {i.attribute.name: i.attribute
              for i in schema.attr_defs if i.attribute.type.value.filters}
-    attrs_filters, name_filters = _parse_filters(filters=filters, attrs=attrs.keys())
+    attrs_filters, entity_filters = _parse_filters(filters=filters, attrs=attrs.keys())
```
Comment (der-gabe) [inline backend/crud.py:441]: Above, this was renamed to `attr_filters` (singular first part).

While it's certainly not necessary, would it make sense to do the same here?

**Review by der-gabe — APPROVED** [review]
While I can't say with 100% certainty that this all does what it's supposed to, I went through all the changes and they all seem to make sense, to me. :+1:

File: backend/crud.py (hunk 1 above)
Comment (crazyscientist) [reply backend/crud.py]: Yes, indeed. I had also considered changing it, but resisted the urge to refactor. But now that you mention it, let me get on it.

**Comment by der-gabe** [comment]
Approved, you can merge. Or should I merge, as the Assignee? I don't know what the process in this project is...

**Comment by crazyscientist** [comment]
Thank you

**Pull request merged**